packages/migrate/src/commands/MigrateStatus.ts

```diff
@@ -165,16 +162,23 @@ ${diagnoseResult.history.unpersistedMigrationNames.join('\n')}`
       //                 - Suggest calling `prisma migrate resolve --applied <migration-name>`
 
       if (listMigrationDirectoriesResult.migrations.length === 0) {
-        return new HowToBaselineError().message
+        console.error(`The current database is not managed by Prisma Migrate.
+        
+Read more about how to baseline an existing production database:
+${link('https://pris.ly/d/migrate-baseline')}`)
+        // Exit 1 to signal that the status is not in sync
+        process.exit(1)
       } else {
         const migrationId = listMigrationDirectoriesResult.migrations.shift() as string
-        return `The current database is not managed by Prisma Migrate.
+        console.error(`The current database is not managed by Prisma Migrate.
 
 If you want to keep the current database structure and data and create new migrations, baseline this database with the migration "${migrationId}":
 ${chalk.bold.greenBright(getCommandWithExecutor(`prisma migrate resolve --applied "${migrationId}"`))}
 
 Read more about how to baseline an existing production database:
-https://pris.ly/d/migrate-baseline`
+https://pris.ly/d/migrate-baseline`)
+        // Exit 1 to signal that the status is not in sync
+        process.exit(1)
```

Comment:
If we can't find a migration table in the db & existing migration files -> exit 1